README.md

```diff
@@ -1,7 +1,114 @@
-# Tour Savvy App
+# Tour Savvy App Frontend
+<a name="readme-top"></a>
```

Comment:
- [ ] Kindly , refactor your functions to ES6 arrow functions so your code complies with the latest standards (I found 4):

![image](https://github.com/Zafron047/tour_savvy_frontend/assets/57421823/b86ab3db-2324-4dc4-b20a-bb7664f8968e)

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

 Shouldn't take you more than 5-10 min so don't worry.  That way, your code will meet the latest standards. 💪

Reply:
@Meltrust 

We have changed all the functions to es6 which is the modern practice. Thank you for improving our code.

👍 😺